app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,25 @@
+<%- if controller_name != 'sessions' %>
+  <%= link_to "Log in", new_session_path(resource_name) %><br />
+<% end -%>
+
+<%- if devise_mapping.registerable? && controller_name != 'registrations' %>
+  <%#= link_to "Sign up", new_registration_path(resource_name) %><br />
```

Comment:
What are our plans related to signing in and users?

This view will eventually need editing once that is decided. There are also currently no translations set in these devise templates yet.